### PR TITLE
Adding 5 languages to the installer (SK, UK, ET, EL, FI)

### PR DESCRIPTION
**Pull request by AlexEven**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The installer is now available in 5 more languages: Slovak, Ukrainian, Estonian, Greek and Finnish
| Type?         | new feature
| Category?     | IN
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Launch the installer, you should see these languages in the language selector and be able to install your shop (translated in these languages) :)

### CODE REVIEW

File: install-dev/langs/el/data/index.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * 2007-2015 PrestaShop
```
Comment (aleeks) [inline install-dev/langs/el/data/index.php]: @AlexEven please, use the good head :(

**Review by aleeks — APPROVED** [review]

**Comment by xBorderie** [comment]
Woohoo!

**Comment by AlexEven** [comment]
Shall we merge? :smirk:

**Comment by aleeks** [comment]
Thank you @AlexEven

**Pull request merged**